src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
Addendum: I am now of the understanding that the new package names will be:

This means the final paths will be:

```yaml
package_name: google.cloud.example_v1
gapic_package_name: google.cloud.example_v1.gapic
proto_package_name: google.cloud.example_v1.proto
```

But only the first will be explicitly configured. Is that accurate?